src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -64,7 +64,16 @@ FilterVariable generateFilter(SnapshotBuilderProgramDataRangeCriteria rangeCrite
                 new Literal(rangeCriteria.getHigh()))));
   }
 
+  FilterVariable selectAllValuesOfListCriteria() {
+    return platform -> "1=1";
+  }
+
   FilterVariable generateFilter(SnapshotBuilderProgramDataListCriteria listCriteria) {
+
+    var isListCriteriaValuesEmpty = listCriteria.getValues().isEmpty();
```

Comment:
are the values ever null? Or just empty?

Reply:
terra-ui, will never pass a `null` but I suppose you can get one from swagger-ui because `SnapshotBuilderProgramDataListCriteria` `values` attribute is set to null on creation. So, I'll check for it! 

Great check!

Reply:
Before changing this, make sure that this can arise by using the swagger API directly, and if possible fix it in the openapi spec rather than adding code to check for this case.